src/commands/dev/index.js

```diff
@@ -35,47 +34,59 @@ const startStaticServer = async ({ settings, log }) => {
   log(`\n${NETLIFYDEVLOG} Server listening to`, settings.frameworkPort)
 }
 
+const NOT_FOUND_EXIT_CODE = 127
+
+const isNonExistingCommandError = (error) => {
+  // `ENOENT` is only returned for non Windows systems
+  // See https://github.com/sindresorhus/execa/pull/447
+  // exitCode === 127 is returned from npm/yarn
+  if (error.code === 'ENOENT' || error.exitCode === NOT_FOUND_EXIT_CODE) {
+    return true
+  }
+
+  // this only works on English versions of Windows
+  return (
+    typeof error.message === 'string' && error.message.includes('is not recognized as an internal or external command')
+  )
+}
+
 const startFrameworkServer = async function ({ settings, log, exit }) {
   if (settings.noCmd) {
     return await startStaticServer({ settings, log })
   }
 
   log(`${NETLIFYDEVLOG} Starting Netlify Dev with ${settings.framework || 'custom config'}`)
-  const commandBin = await which(settings.command).catch((error) => {
-    if (error.code === 'ENOENT') {
-      throw new Error(
-        `"${settings.command}" could not be found in your PATH. Please make sure that "${settings.command}" is installed and available in your PATH`,
+
+  const commandWithArgs = `${settings.command} ${settings.args.join(' ')}`
+  // we use reject=false to avoid rejecting synchronously when the command doesn't exist
+  // we can't try->await->catch since we don't want to block on the framework server which
+  // is a long running process
+  const frameworkProcess = execa.command(commandWithArgs, { preferLocal: true, reject: false })
+  frameworkProcess.stdout.pipe(stripAnsiCc.stream()).pipe(process.stdout)
```

Comment:
Just double-checking that we really need to strip ANSI sequences here?
Otherwise, we could just use `stdio: 'inherit'`, which would be simpler and more robust.

Reply:
Seems like this was a [bug fix](https://github.com/netlify/cli/pull/828) for multiple bugs.
I'll check what happens without it.

Reply:
Would it be worth checking in a different PR?

Reply:
Yes! See https://github.com/netlify/cli/issues/2879